versioned_docs/version-8.7/components/camunda-integrations/sap/odata-connector.md

```diff
@@ -126,6 +187,94 @@ The result of any query, whether it is reading or writing to the SAP system, is
 
 The query result can either be mapped to a single result variable or worked on [via FEEL with an expression](/components/connectors/use-connectors/index.md#result-expression). The same is applicable to `getResponse`, as a result variable contains the described query JSON in its entirety. The result expression `{getStatusCode: statusCode}` would only hold the HTTP status code in the `getStatusCode` process variable.
 
+For `$batch` requests, the query result is an Array of the above mentioned result structure. This is an example for an OData v2 Batch Request, consisting of one Read and one Update operation:
+
+```json
+{
+  "<your output mapping result variable>": [
+    {
+      "result": {
+        "d": {
+          "__metadata": {
+            "id": "<host>/sap/opu/odata/sap/API_BUSINESS_PARTNER/A_BusinessPartner('...')",
+            "uri": "<host>/sap/opu/odata/sap/API_BUSINESS_PARTNER/A_BusinessPartner('...')",
+            "type": "API_BUSINESS_PARTNER.A_BusinessPartnerType"
+          },
+          "FirstName": "..."
+        }
+      },
+      "statusCode": 200
+    },
+    { "result": null, "statusCode": 204 }
+  ]
+}
+```
+
+In case one of the operations in the Batch request fails, the error is relayed to the `result` node for the request, as in the second request in this sample result where the Business Partner with number `0000000` (obviously) couldn't be found.
+
+```jsonc
```

Comment:
Should this be json or jsonc?

Reply:
`jsonc` allows comments in JSON, which adds to readability. 
If only `json` is possible, then I'm fine with this too.